lib/cm3/nvic.c

```diff
@@ -143,6 +143,7 @@ uint8_t nvic_get_irq_enabled(uint8_t irqn)
  *
  * There are 4 priority levels only, given by the upper two bits of the
  * priority byte, as required by ARM standards. No grouping available.
+ * Only word access allowed!
```

Comment:
given that people are using the api for this, this comment is meaningless for them.

Reply:
Unclear wording. Is with regard to the Cortex-M registers, not the API access.